### PR TITLE
Trigger incoming call via cache primer

### PR DESCRIPTION
**Pull request by julianjelfs**

### CODE REVIEW

File: frontend/openchat-client/src/openchat.ts [hunk 1]
```diff
@@ -3112,7 +3111,7 @@ export class OpenChat extends OpenChatAgentWorker {
                         threadRootMessageIndex,
                     });
                 }
-                if (unconfirmed.delete(context, messageId)) {
+                if (unconfirmed.contains(context, messageId)) {
```
Comment (hpeebles) [inline frontend/openchat-client/src/openchat.ts:3114]: This has reverted my revert

Comment (julianjelfs) [reply frontend/openchat-client/src/openchat.ts]: Well spotted - not sure how that happened.

**Review by hpeebles — APPROVED** [review]

**Comment by hpeebles** [comment]
Are we checking if the chat is muted anywhere? We probably shouldn't ring for chats that are muted.

**Comment by julianjelfs** [comment]
> Are we checking if the chat is muted anywhere? We probably shouldn't ring for chats that are muted.

We aren't at the moment. I'm in two minds about that. It's only for private chats. And I think there's probably be a difference between not wanting notifications and not wanting to receive calls.

**Review by hpeebles — APPROVED** [review]

**Pull request merged**